packages/govern-server/src/core/data/TheGraphWrapper.ts

```diff
@@ -0,0 +1,70 @@
+import fetch from 'isomorphic-unfetch'
+import { Client } from '@urql/core'
+import { DocumentNode } from 'graphql'
+import { ErrorConnection } from '../errors'
+import { QueryResult } from '../types'
+
+type TheGraphWrapperOptions = {
+  verbose?: boolean
+}
+
+export default class TheGraphWrapper {
```

Comment:
A name like ``TheGraphAdapter`` or ``TheGraphClient`` could be more self-describing. 🤷‍♂️

Reply:
Hint: As discussed with Pierre should the export of a class be on the last line of a file. :-)